### ISSUE TITLE
Alignment of bullet points for image-steps in https://cyf-curriculum.netlify.app/guides/deployment-netlify/first-site/

### ISSUE THREAD

**Issue by illicitonion** [CodeYourFuture/curriculum#72]
**Describe the bug**

On https://cyf-curriculum.netlify.app/guides/deployment-netlify/first-site/ I'd expect the bullet points to be towards the top of the images in the steps, or corresponding to some text before the screenshots.

Instead, they float at kind of strange positions relative to the images, e.g.

<img width="816" alt="Screenshot 2023-06-11 at 20 14 34" src="https://github.com/CodeYourFuture/curriculum/assets/1131704/2d7cdd99-cbbc-460c-a5c5-aec88d135b39">

Bullet point 4 is really far down the step being done...

**Comment by SallyMcGrath** [comment]
Yeah this is the default behaviour for increment markers. You can have list-style-position: inside, which will put them on a blank line at the top, or we can remake the ::marker with a custom counter, but it's a bit of a faff. 

I don't really mind either way on this one.

**Comment by illicitonion** [comment]
I don't love the blank line in `list-style-position: inside;` but I think it's probably nicer than what we have now - I'm not sure where the best place to put this style would be, I guess assets/styles/03-elements/base.scss or assets/styles/layout/main.scss? Or a new assets/styles/03-elements/lists.scss?

**Comment by SallyMcGrath** [comment]
once it gets faffy, break it out into lists.scss (probably should also break out details soon)
